web/src/buddy_data.js

```diff
@@ -1,3 +1,5 @@
+import $ from "jquery";
```

Comment:
@timabbott how do you feel about using jquery in this file? I wasn't sure how else to set the placeholder text only when search changed, while also not causing circular imports

Reply:
It's definitely an abstraction boundary violation. But we can plan to clean that up in a follow-up. Please add a "TODO" comment next to `set_is_searching_users` indicating that it should be in X module but can't be easily placed there due to import cycles.

Reply:
Done!